client/sites/tools/staging-site/components/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -421,9 +421,18 @@ const SyncCardContainer = ( {
 							status="is-error"
 							icon="mention"
 							showDismiss={ false }
-							text={ translate( 'We couldn’t synchronize the %s environment.', {
-								args: [ siteToSync ?? '' ],
-							} ) }
+							text={
+								error === 'studio_import_in_progress'
+									? translate(
+											'We couldn’t synchronize the %s environment. Studio push operation is currently in progress.',
+											{
+												args: [ siteToSync ?? '' ],
```

Comment:
There’s a mistake in the typing; let’s fix it and make the code clearer:
```suggestion
												args: [ siteToSync ],
```

Reply:
Remove `null` here https://github.com/Automattic/wp-calypso/pull/98216/files#diff-f9f4922cb3e731995f5a71b06ba9a4655b30c9eba9b62d36e81df7a8e630d30fR354

Reply:
Also, you can remove optional chaining inside this condition `{ error && ! isJetpackConnectionError && ! isFailedSyncError && (`

Reply:
Also we can clean up this module `client/sites/tools/staging-site/components/staging-site-card/card-content/staging-site-sync-loading-bar-card-content.tsx` with removing `} else {`, it's redundant.

Reply:
I understand that the mistake was before, but supporting the mistake doesn't look nice, so let's clean up the code a bit :)

Reply:
Great catch! Thanks, Vova!